src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationMessageProcessor.java

```diff
@@ -48,6 +49,8 @@ public boolean processNextMessage() throws ServiceBusException, InterruptedExcep
             } catch (InvalidMessageException ex) {
                 log.error("Invalid notification message with ID: {} ", message.getMessageId(), ex);
                 finaliseProcessedMessage(message, MessageProcessingResult.UNRECOVERABLE_FAILURE);
+            } catch (DuplicateMessageIdException ex) {
+                handleDuplicateMessageId(message);
```

Comment:
the trace (and the message within exception is gone (which has db id in there for a reference)). perhaps need to pass message to referred method here? i doubt need any trace so that message should be enough in my opinion

Reply:
Updated to pass the exception message and log it